code/modules/projectiles/updated_projectiles/gun_system.dm

```diff
@@ -1005,6 +1007,10 @@ and you're good to go.
 	return TRUE
 
 /obj/item/weapon/gun/proc/simulate_scatter(obj/item/projectile/projectile_to_fire, atom/target, turf/targloc, total_scatter_chance = 0, mob/user, burst_scatter_bonus = 0)
+	if(isnull(targloc)) //For debugging only.
```

Comment:
:eyes:

Reply:
Happened once. But not since then. Likely due to an earlier runtime, but who knows?
The thing is now filled with debug messages because there's sometimes quirky behavior whose causes I haven't yet narrowed down.